pkg/event/processor.go

```diff
@@ -33,7 +33,7 @@ import (
 
 // Processor processes events
 type Processor interface {
-	ProcessEvent(event UserEvent)
+	ProcessEvent(event UserEvent) bool
```

Comment:
Is the bool return value ever used? I don’t see why this is necessary unless the client makes use of it

Reply:
👍